lib/bolt/target.rb

```diff
@@ -85,6 +87,18 @@ def to_s
       "Target('#{@uri}', #{opts})"
     end
 
+    def to_h
+      options.merge(
+        'name' => name,
+        'uri' => uri,
+        'protocol' => protocol,
+        'user' => user,
+        'password' => password,
```

Comment:
We redacted password in `to_s`, should we also redact it here?

Reply:
no that would break serialization.